bot/bot.go

```diff
@@ -185,12 +185,20 @@ func (bot *Bot) messageHandler(s *discordgo.Session, m *discordgo.MessageCreate)
 			bot.bumpStatistics(moved)
 
 		case "summon":
-			_, _ = bot.Summon(m, params)
+			moved, err := bot.Summon(m, params)
+			if err != nil {
 
-		case "help":
-			_, _ = s.ChannelMessageSend(m.ChannelID, fmt.Sprintf(bot.Messages[lang]["HelpMessage"], bot.Prefix, bot.Prefix, bot.Prefix))
+				return
+			}
+			bot.bumpStatistics(moved)
```

Comment:
bumping statistics should be done bythe bot.Summon and not by the messageHandler!

Reply:
Currently the number of moved users is returned by both `Move` and `Summon` commands, and the statistics bump happens in only one line. If this step was moved into the commands, it would have to happen in 2 different lines in each command (61 and 82 for `summon`, 55 and 88 for 'move)

Reply:
Even so, they should be moved to their respective commands to keep things organized!
Let's make that a new change for the future!